backend/src/main/resources/api.yaml

```diff
@@ -500,6 +522,36 @@ components:
           example: Securepassword1337!
     # ==OTHER SCHEMAS==
     # Utility
+    JsonPatchRequest:
+      type: array
+      items:
+        $ref: "#/components/schemas/JsonPatchOperation"
+    JsonPatchOperation:
+      type: object
+      description: A JSONPatch document as defined by RFC 6902.
+      required:
+        - "op"
+        - "path"
+      properties:
+        op:
+          type: string
+          description: The operation to be performed.
+          enum:
+            - "add"
+            - "remove"
+            - "replace"
+            - "move"
+            - "copy"
+            - "test"
+        path:
+          type: string
+          description: A JSON-Pointer.
+        value:
+          type: object
+          description: The value to be used within the operations.
+        from:
+          type: string
+          description: A string containing a JSON Pointer value.
```

Comment:
Czy jest możliwość dodać tutaj `example`? Albo chociaż później wyjaśnij co i jak ma być wysyłane, bo może po prostu jest za późno jak to oglądam, ale nie zaczaiłem jeszcze jakoś 🤷🏻‍♂️